### ISSUE TITLE
Turn on `emoji` will cause hyperlinks containing two colons to fail

### ISSUE THREAD

**Issue by DIYer22** [simov/markdown-viewer#205]
For example:
```markdown
- [public typing demo](https://megvii-my.sharepoint.cn/:v:/g/personal/yanglei_megvii_com/EVdE-isEsd1Jt9NiJ4NreigBUKDCK64i04erQnnE-qKvAQ?e=9DQG26)
```
Yes, this is a real web hyperlink containing two colons 🤣

The render result:
![image](https://user-images.githubusercontent.com/10448025/215343491-0935ea86-bf9b-4ec7-96eb-f3c9ee69d14f.png)

**Comment by simov** [comment]
That's unfortunate, though I think I might be able to guard against certain HTML tags, since the emojis are being applied on top of the already rendered HTML string using this very simple regular expression `:[a-z0-9_+-]+:`.

**Comment by simov** [comment]
This was fixed in v5.1, emojis inside HTML tag strings and inside `<code>` blocks are being ignored now.
